web-api/src/app.ts

```diff
@@ -861,12 +861,12 @@ app.get('/sections/:section/judge', lambdaWrapper(getJudgeInSectionLambda));
     lambdaWrapper(getTrialSessionsForJudgeActivityReportLambda),
   );
   app.post(
-    '/judge-activity-report/opinions',
-    lambdaWrapper(getOpinionsFiledByJudgeLambda),
+    '/async/judge-activity-report/opinions',
```

Comment:
Why did these endpoints need to become async?

Reply:
Processing or orders and opinions for `All Judges` for 2+ years exhaust the 29 seconds timeout for regular lambdas. If  we do end up limiting the search to just one year, this may no longer be necessary.

Reply:
Ok that does make sense, thanks for explaining! I am still a bit curious, is there anything that can be done to make that endpoint more efficient? It appears as if it's doing something very similar to what advanced opinion search does when you submit a search for all judges without specifying a date range, although I could be wrong.

I see that the `getOpinionsFiledByJudgeInteractor` is using a Promise.all to query elasticsearch for each judge in the case that 'all judges' is selected. Would it be more performant to query for literally all judges and then filter out legacy judges from the results? Again, I could be totally wrong, just trying to think of a way to make the request more performant for a user.

Reply:
Agree with you @rachelschneiderman . We are going to pursue improvements to performance, perhaps in another user story. I believe if we made use of an `aggs` query, we can move this back to a synchronous request. It's a more involved `aggs` query because we need to check if the case does not have certain documents filed, but I think that's doable with the parent/child relationship they have in ES.